src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -74,48 +75,50 @@ def load_arguments(self, _):
         c.argument('id_', options_list=['--id'], help='Id of the key to be regenerated. Can be found using az appconfig credential list command.')
 
     with self.argument_context('appconfig kv import') as c:
-        c.argument('label', help="Imported KVs will be assigned with this label. If no label specified, will assign null label.")
-        c.argument('prefix', help="This prefix will be appended to the front of imported keys.")
-        c.argument('source', options_list=['--source', '-s'], choices=['file', 'appconfig', 'appservice'], validator=validate_import, help="The source of importing.")
+        c.argument('label', help="Imported KVs and feature flags will be assigned with this label. If no label specified, will assign null label.")
+        c.argument('prefix', help="This prefix will be appended to the front of imported keys. Not applicable for feature flags.")
+        c.argument('source', options_list=['--source', '-s'], choices=['file', 'appconfig', 'appservice'], validator=validate_import, help="The source of importing. Note that importing feature flags from appservice is not supported.")
```

Comment:
Here `choices` is not recommended that you use this because the choice lists will be case sensitive. Instead see ["Registering Enums"](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-enums).

Reply:
done

---
In reply to: [355225688](https://github.com/Azure/azure-cli/pull/11467#discussion_r355225688) [](ancestors = 355225688)